uwac/libuwac/uwac-input.c

```diff
@@ -122,10 +122,13 @@ static UwacReturnCode set_cursor_image(UwacSeat* seat, uint32_t serial)
 			break;
 	}
 
+	int buffer_add_listener_success = -1;
```

Comment:
maybe move variable declaration to beginning of scope to satisfy older compilers